CSharpSource/Source/api/Leaderboard/LeaderboardResult.cs

```diff
@@ -139,14 +141,14 @@ internal struct LEADERBOARD_RESULT
             [MarshalAs(UnmanagedType.SysInt)]
             public IntPtr Columns;
 
-            [MarshalAs(UnmanagedType.U4)]
-            public Int32 ColumnsSize;
+            [MarshalAs(UnmanagedType.SysInt)]
+            public IntPtr ColumnsSize;
```

Comment:
did leaderboard_c.h change?

Reply:
Why IntPtr and not just Int

---
In reply to: [154735620](https://github.com/Microsoft/xbox-live-unity-plugin/pull/160#discussion_r154735620) [](ancestors = 154735620)